Homework 2

petrovromanval opened the pull request.

petrovromanval committed:
Сделано домашнее задание №2

petrovromanval committed:
Добавлен файл README.md с описанием домашнегго задания №2

petrovromanval committed:
Отредактировал текст в README.md

petrovromanval committed:
После проверки на валидность добавлен атрибут   lang="ru" в тег <html>

AlexMubarakshin commented on homework_2/FenderStratocaster.html:
```diff
@@ -0,0 +1,115 @@
+
+            <p>
+                <label>
+                    Ваше E-mail<br><input type="text" placeholder="Введите ваш E-mail"> 
```
Используй для ввода email, input с типом 'email'

petrovromanval replied:
Исправил

AlexMubarakshin commented on homework_2/FenderStratocaster.html:
```diff
@@ -0,0 +1,115 @@
+
+            <p>
+                <label>
+                    Контактный телефон<br><input type="text" placeholder="+7 XXX XXXXXXX"> 
```
Аналогично как и с email

petrovromanval replied:
Исправил

AlexMubarakshin commented on homework_2/FenderStratocaster.html:
```diff
@@ -0,0 +1,115 @@
+
+            <input type="reset" value="Очистить">
+            <input type="submit" value="Отправить">
+            <input type="button" value="Кнопка">
```
"Кнопка"? 🧐😀

petrovromanval replied:
Просто она пока ни к какому действию не привязана, во я ее и назвал просто "Кнопка"))).  Сделал ее, потому что в ДЗ было написано использовать тег button. Я ее удалю)

AlexMubarakshin reviewed (changes requested):
Оставил пару комментов в html'ке по типам input'ов.
Необходимо их исправить 😊

petrovromanval committed:
Исправил в html типы input для поля email на type ="email", для поля телефон на type="tel".
Так как для форм в качестве селекторов я ипользовал атрибуты, поэтому в css пришлось добавить свойства для этих селекторов.

AlexMubarakshin commented on homework_2/FenderStratocaster.html:
```diff
@@ -103,8 +103,7 @@ <h2>Формы обратной связи</h2>
 
             <input type="reset" value="Очистить">
```
Про тег button имелось что его можно (И даже необходимо) использовать вместо:
```html
<input type="submit" value="Отправить">
```

вот это:
```html
<button type="submit">Отправить</button>
```

😊

petrovromanval replied:
Спасибо, что поправил, а то я чет вместо тега button использовал type="button" в input. Исправил и сделал commit №6

petrovromanval committed:
Поменял input c кнопками на button.

AlexMubarakshin committed:
Merge branch 'master' into homework_2
